Adds tests of parameters

johnlees opened the pull request:
Closes #5

Attempt at some sensible unit tests for this, not bothering to check the passed through parameters, or stuff generated by functions that's already in the existing test.

Suggestions very welcome!

richfitz commented on tests/testthat/test-parameters.R:
```diff
@@ -29,3 +29,72 @@ test_that("Parameters are generated as before", {
+  for (class_idx in 1:length(trans_increase)) {
+    expect_true(all(test_params$trans_increase[,class_idx] == trans_increase[class_idx]))
+  }
+  for (class_idx in 1:length(trans_increase)) {
```
perhaps
```
expect_equal(test_params$trans_profile[,class_idx], trans_profile[class_idx])
```

johnlees commented:
Fixed those loops and added new messy severity files to hit the other lines

The only line not being hit is not hittable, with the current code and function arguments. Perhaps we should consider removing `support.R` now we are getting population from the severity file.
